sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -43,6 +46,7 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
     private readonly studentAssessmentService: StudentAssessmentService,
     private readonly studentRestrictionService: StudentRestrictionService,
     private readonly notificationActionsService: NotificationActionsService,
+    private readonly studentRestrictionsService: StudentRestrictionsService,
```

Comment:
is the StudentRestrictionsService and the one StudentRestrictionService on line 46 are same? if yes can we please remove the unused service, in both the services and here.

Reply:
it's not. it's different. changed the name as per our discussion